src/components/Animal/Animal.jsx

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+
+export default function Spot({ spots, animal, organizers }) {
+	return (
+		<article>
+			<header>
+				<h2>{animal.name}</h2>
+			</header>
+
+			<section>
+				<h3>Description</h3>
+				<p>{animal.description}</p>
+			</section>
+
```

Comment:
I would delete empty lines inside a return statement (I think Prettier normally would).

Reply:
I know, but as long as we don't have a corresponding explicit requirement for our code, I prefer to keep my extra spacing to have a better overview.